libsolidity/AST.h

```diff
@@ -440,13 +442,22 @@ class FunctionDefinition: public Declaration, public VariableScope, public Docum
 class VariableDeclaration: public Declaration
 {
 public:
-	VariableDeclaration(SourceLocation const& _location, ASTPointer<TypeName> const& _type,
-							ASTPointer<ASTString> const& _name, ASTPointer<Expression> _value,
-							Visibility _visibility,
-							bool _isStateVar = false, bool _isIndexed = false):
-			Declaration(_location, _name, _visibility),
-			m_typeName(_type), m_value(_value),
-			m_isStateVariable(_isStateVar), m_isIndexed(_isIndexed) {}
+	VariableDeclaration(
+			SourceLocation const& _location,
```

Comment:
We need a clear distinction between parameters and initializer list. Therefore I would say either single tab and `)` unindented on the next line or the way it is here.